rbac/management/group/view.py

```diff
@@ -382,9 +382,19 @@ def remove_principals(self, group, principals, account=None, org_id=None):
 
             for username in principals:
                 try:
-                    principal = Principal.objects.get(username__iexact=username, tenant=tenant)
+                    principal = Principal.objects.get(username__iexact=username, tenant=tenant, group=group)
                 except Principal.DoesNotExist:
                     logger.info("No principal %s found for org id %s.", username, org_id)
+                    return {
+                        "status_code": status.HTTP_404_NOT_FOUND,
+                        "errors": [
+                            {
+                                "detail": f"User '{username}' not found in the group '{group.name}'.",
+                                "status": "404",
```

Comment:
nitpick: instead of hard coding 404 can we reference `status.HTTP_404_NOT_FOUND` like above? just looks a little cleaner imo

Reply:
good catch 👍